frontend/tests/builders/partyBuilder.js

```diff
@@ -30,13 +30,20 @@ export class PartyBuilder {
   }
 
   build() {
-    if (this.players === []) {
+    if (this.players && this.players.length === 0) {
```

Comment:
with `this.players` being initialised to `[]`, shouldn't this be exactly the same as `this.players === []`?

Reply:
Yes. I don't know why, but it didn't work.
Even with debugging output was always `[]`. I'm happy if you could take a look becasue this bugged the hell out of me!

Reply:
Turns out I was wrong. Stupid JavaScript...

```
$ node
Welcome to Node.js v14.15.4.
Type ".help" for more information.
> [] === []
false
```